src/controllers/route.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+const home = require('../views/home');
+
+router.get('/', (req, res) => {
+   res.render('home', { activePage: {home: true}});
+});
+
```

Comment:
I figured that, so deleted that footer comment, but apparently not fast enough :smile: 

Is the fix for res.render being pushed to the 'paul2' branch so I can merge it?